packages/shortcuts-extension/test/shortcuts.spec.ts

```diff
@@ -309,7 +309,154 @@ describe('@jupyterlab/shortcut-extension', () => {
       const shortcuts = (await settings.get('shortcuts')
         .composite) as ISettingRegistry.IShortcut[];
 
-      expect(shortcuts).toHaveLength(1);
+      expect(shortcuts).toHaveLength(5);
     });
   });
+
+  it('should add punctuation aria-label @A11y', async () => {
+    const bar: ISettingRegistry.IPlugin = {
+      data: {
+        composite: {},
+        user: {}
+      },
+      id: pluginId,
+      raw: '{}',
+      schema: {
+        'jupyter.lab.shortcuts': [
+          {
+            command: 'application:activate-previous-tab',
+            keys: ['Ctrl Shift ['],
+            selector: '.jp-contextualShortcut-key'
+          },
+          {
+            command: 'settingeditor:open',
+            keys: ['Ctrl ,'],
+            selector: '.jp-contextualShortcut-key'
+          }
+        ],
+        type: 'object'
+      },
+      version: 'test'
+    };
+
+    const connector: IDataConnector<
+      ISettingRegistry.IPlugin,
+      string,
+      string,
+      string
```

Comment:
```suggestion
      string
```

The last two are not needed, see changes in https://github.com/jupyterlab/jupyterlab/pull/16043